src/store.rs

```diff
@@ -140,26 +137,88 @@ impl fmt::Display for Error {
 /// Prefer these methods when constructing literal data or assembling program fragments in
 /// tests or during evaluation, etc.
 impl<F: LurkField> Store<F> {
-    pub fn nil(&mut self) -> Ptr<F> {
-        self.lurk_sym("nil")
+    pub fn expect_constants(&self) -> &NamedConstants<F> {
+        self.constants
+            .get()
+            .expect("Constants must have been set during instatiation")
+    }
+
+    pub fn nil_ptr(&self) -> Ptr<F> {
+        self.expect_constants().nil.ptr()
+    }
+
+    pub fn cons_ptr(&self) -> Ptr<F> {
+        self.expect_constants().cons.ptr()
+    }
+
+    pub fn cdr_ptr(&self) -> Ptr<F> {
+        self.expect_constants().cdr.ptr()
+    }
+
+    pub fn begin_ptr(&self) -> Ptr<F> {
+        self.expect_constants().begin.ptr()
+    }
+
+    pub fn eq_ptr(&self) -> Ptr<F> {
+        self.expect_constants().equal.ptr()
     }
 
-    pub fn t(&mut self) -> Ptr<F> {
-        self.lurk_sym("t")
+    pub fn quote_ptr(&self) -> Ptr<F> {
+        self.expect_constants().quote.ptr()
     }
 
+    pub fn t_ptr(&self) -> Ptr<F> {
+        self.expect_constants().t.ptr()
+    }
+
+    pub fn dummy_ptr(&self) -> Ptr<F> {
+        self.expect_constants().dummy.ptr()
+    }
+
+    pub fn let_ptr(&self) -> Ptr<F> {
+        self.expect_constants().let_.ptr()
+    }
+
+    pub fn letrec_ptr(&self) -> Ptr<F> {
+        self.expect_constants().letrec.ptr()
+    }
+
+    pub fn current_env_ptr(&self) -> Ptr<F> {
+        self.expect_constants().current_env.ptr()
+    }
+
+    pub fn open_ptr(&self) -> Ptr<F> {
+        self.expect_constants().open.ptr()
+    }
+
+    pub fn secret_ptr(&self) -> Ptr<F> {
+        self.expect_constants().secret.ptr()
+    }
+
+    pub fn num_ptr(&self) -> Ptr<F> {
+        self.expect_constants().num.ptr()
+    }
+
+    pub fn lambda_ptr(&self) -> Ptr<F> {
+        self.expect_constants().lambda.ptr()
+    }
```

Comment:
In all good humor, I am quite horrified by all these `Store` methods owing their existence only to the coincidental existence of some Lurk symbols. The mere fact of the 'constants' seems to already trespass as far in that direction as is decent — so effectively 'unencapsulating' the enumeration seems a bit perverse. If you want a convenient way to grab all these symbols, could you use a macro? Like `whatever!(lambda)`?